src/handlers/applications.js

```diff
@@ -1,14 +1,86 @@
-const applicationService = require('../services/applications')
+const service = require('../services/applications')
+const schemas = require('../schemas/applications')
+const response = require('../utils/response')
+const logger = require('../utils/logger')('application_handler')
 
 async function create (event) {
-  const { body: rawBody } = event
-  const body = JSON.parse(rawBody)
+  try {
+    const { body: rawBody } = event
+    const body = JSON.parse(rawBody)
+
+    const payload = await schemas.createApplications.validate(
+      body,
+      { abortEarly: false }
+    )
+
+    const application = await service.create(payload)
+
+    return response.sucess({
+      body: application,
+      statusCode: 201,
+    })
+  } catch (error) {
+    if (error.name === 'ValidationError') {
+      return response.error({
+        statusCode: 400,
+        message: error.errors.toString(),
+      })
+    }
+
+    return response.error() 
```

Comment:
Codacy has a fix for the issue: [Missing semicolon.](https://app.codacy.com/gh/lucianopf/serverless-webhooks-service/pullRequest?prid=9084646)
```suggestion
    return response.error(); 
```